docs/meshstack.customizing.md

```diff
@@ -5,18 +5,6 @@ title: Customizing
 
 meshStack allows operators to customise the user experience according to the needs of their organisation.
 
-## Feedback Mailbox
```

Comment:
Is this an unrelated change? Or is that because we replaced it with a link to canny instead?

Reply:
Yes I ran into it and we don't offer this mailbox anymore in favor of Canny